.teamcity/src/main/kotlin/configurations/PerformanceTestsPass.kt

```diff
@@ -55,13 +55,18 @@ class PerformanceTestsPass(model: CIBuildModel, performanceTestProject: Performa
 subprojects/$performanceProjectName/build/performance-test-results.zip
 """
     if (performanceTestProject.performanceTests.any { it.testProjects.isNotEmpty() }) {
+        val performanceTestBuildIds = performanceTestProject.performanceTests
+            .filter { it.testProjects.isNotEmpty() }
+            .joinToString(",") { "%dep.${it.id}.env.BUILD_ID%" }
+
         gradleRunnerStep(
             model,
             ":$performanceProjectName:$taskName --channel %performance.channel%",
             extraParameters = listOf(
                 "-Porg.gradle.performance.branchName" to "%teamcity.build.branch%",
                 "-Porg.gradle.performance.db.url" to "%performance.db.url%",
-                "-Porg.gradle.performance.db.username" to "%performance.db.username%"
+                "-Porg.gradle.performance.db.username" to "%performance.db.username%",
+                "-Porg.gradle.performance.performanceTestBuildIds" to performanceTestBuildIds
```

Comment:
```suggestion
                "-Porg.gradle.performance.dependencyBuildIds" to performanceTestBuildIds
```

Nit: Performance once in the name should be enough.